internal/deputil/url_test.go

```diff
@@ -15,15 +15,56 @@
 package deputil
 
 import (
+	"fmt"
+	"net/http"
 	"testing"
 
+	"github.com/slackapi/slack-cli/internal/slackdeps"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
 )
 
 func Test_URLChecker(t *testing.T) {
-	url := URLChecker("https://github.com/slack-samples/deno-starter-template")
-	assert.Equal(t, "https://github.com/slack-samples/deno-starter-template", url, "should return url when url is valid")
+	tests := map[string]struct {
```

Comment:
note: Reimplemented the tests as a table test and extended the cover. This function now has 100% test coverage. 💯